Geometry/RPCGeometryBuilder/plugins/RPCGeometryESModule.cc

```diff
@@ -33,14 +28,11 @@ RPCGeometryESModule::RPCGeometryESModule(const edm::ParameterSet& p)
       useDDD_{p.getUntrackedParameter<bool>("useDDD", true)},
       useDD4hep_{p.getUntrackedParameter<bool>("useDD4hep", false)} {
   auto cc = setWhatProduced(this);
-
   const edm::ESInputTag kEmptyTag;
-  if (useDDD_) {
+  if (useDDD_ || useDD4hep_) {
```

Comment:
I think these lines have to be reverted. You can't use the old `DDCompactView` with DD4hep.

Reply:
Done